javascript/javascript3/week3/Canvas/draw-circles.js

```diff
@@ -0,0 +1,105 @@
+console.log("loadeddd");
+//USING JS
+let canvas = document.getElementById("myCanvas");
+
+let context = canvas.getContext("2d");
+
+function drawSeveralCircles() {
+    // Draw several circles
+    for (var i = 0; i < 10; i++) {
```

Comment:
Var is not good practice to use. As per ES6 specifications use either const or let, depending on the variable used.